swagger/aggregator.yml

```diff
@@ -0,0 +1,81 @@
+openapi: 3.0.1
+info:
+  title: XAIN FL Aggregator API
+  contact:
+    email: engineering@xain.io
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.0
```

Comment:
This should be `0.0.7`. But I'm afraid we forget to update that when bumping the versions. Perhaps we should have a script.for this.